selfdrive/car/volkswagen/values.py

```diff
@@ -302,7 +302,7 @@ class CAR(Platforms):
   )
   TROC_MK1 = VolkswagenMQBPlatformConfig(
     "VOLKSWAGEN T-ROC 1ST GEN",  # Chassis A1
-    VWCarInfo("Volkswagen T-Roc 2018-22", footnotes=[Footnote.VW_MQB_A0]),
+    VWCarInfo("Volkswagen T-Roc 2018-23"),
```

Comment:
@jyoung8607 does this look correct to you?

Reply:
The model-year can be extended, but I'm not comfortable with removing the footnote. We still don't have a good handle on when/why MQB-A0s get the combo BCM/gateway. There's some evidence it may be a region/market thing.

Reply:
Oh, I didn't consider futher about worldwide, just saw the catalog print TROC is MQB-A1. 
Shall I close this PR and re-PR again？

Reply:
I checked and you are correct, T-Roc isn't MQB-A0. In that case, your PR is valid. Thank you for the fix!

@sshane I'm not sure that @Saber422 is set up to regen CARS.md, can you do that?